### DISCUSSION TITLE
Amazing project

### DISCUSSION THREAD

**Discussion by bernky** [kovidgoyal/kitty#3184] [General]
Thank you @kovidgoyal  and all contributors... This and calibre have improved my life and many others in so many ways :-) 

 The fact this is the only terminal emulator I could find that has a easy config syntax and It also handles ligatures like a swiss knife! Makes Kitty just as more awesome!

**Comment by kovidgoyal** [comment]
You are most welcome :+1:

**Comment by apiraino** [comment]
I also would like to thank @kovidgoyal for this amazing project and the outstanding support he's providing to the users.

**Comment by jef** [comment]
Also came to post this! The sheer amount of maintenance and feature work done here by @kovidgoyal and contributors is extraordinary.
